### PR TITLE
Updated dcos node list for node draining.

### PR DESCRIPTION
**Pull request by armandgrillet**

### CODE REVIEW

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

File: pkg/cmd/node/node_list.go [hunk 1]
```diff
@@ -151,21 +178,24 @@ func newCmdNodeList(ctx api.Context) *cobra.Command {
...
-					m.IP,
-					strings.Join(m.PublicIPs, ", "),
-					tablewriter.ConditionString(m.ID != "", m.ID, "N/A"),
-					m.Type,
-					tablewriter.ConditionString(m.Region != "", m.Region, "N/A"),
-					tablewriter.ConditionString(m.Zone != "", m.Zone, "N/A"),
-				}
-				for _, field := range fields {
-					tableItem = append(tableItem, nodeExtraField(m, strings.Split(field, ".")))
+				if mesosID == "" || (mesosID != "" && mesosID == m.ID) {
```
Comment (bamarni) [inline pkg/cmd/node/node_list.go]:
```suggestion
				if mesosID == "" || mesosID == m.ID {
```

File: pkg/cmd/node/node_list.go [hunk 2]
```diff
@@ -94,45 +113,53 @@ func newCmdNodeList(ctx api.Context) *cobra.Command {
...
-						s.TaskError = sSummary.TaskError
-						s.TaskFailed = sSummary.TaskFailed
-						s.TaskFinished = sSummary.TaskFinished
-						s.TaskKilled = sSummary.TaskKilled
-						s.TaskKilling = sSummary.TaskKilling
-						s.TaskLost = sSummary.TaskLost
-						s.TaskRunning = sSummary.TaskRunning
-						s.TaskStaging = sSummary.TaskStaging
-						s.TaskStarting = sSummary.TaskStarting
+				if mesosID == "" || (mesosID != "" && mesosID == s.ID) {
```
Comment (bamarni) [inline pkg/cmd/node/node_list.go]:
```suggestion
				if mesosID == "" || mesosID == s.ID {
```

**Pull request merged**